lib/Jejik/MT940/Parser/AbstractParser.php

```diff
@@ -120,25 +132,26 @@ protected function splitStatements($text)
      * is at offset 0 and the description line text (if any) at offset 1.
      *
      * @param string $text Statement text
+     *
      * @return array Nested array of transaction and description lines
      */
-    protected function splitTransactions($text)
+    protected function splitTransactions($text): array
     {
-        $offset = 0;
-        $length = 0;
-        $position = 0;
-        $transactions = array();
+        $offset       = 0;
```

Comment:
Personally I am not a fan of multi-line alignment like this. But if you insist...

Reply:
The line was so long ... ;)